source/manuals/accessibility.html.md.erb

```diff
@@ -1,13 +1,13 @@
 ---
 title: Building accessible services
-last_reviewed_on: 2023-03-08
+last_reviewed_on: 2023-09-21
 review_in: 6 months
 owner_slack: '#accessibility-community'
 ---
 
 # <%= current_page.data.title %>
 
-Government services must be accessible to everyone. This includes anyone with a visual, hearing, speech, motor or learning impairment. This also includes anyone with a temporary or situational disability, such as a person with a broken arm or working in a loud environment.
+Government services must be accessible to everyone. This includes anyone with a visual, hearing, speech, motor, learning or cognitive impairment. This also includes anyone with a temporary or situational disability, such as a person with a broken arm or working in a loud environment.
```

Comment:
Aren't learning impairments included in cognitive impairments?

Reply:
I thought this as well and still don't fully understand why learning impariments wouldn't be included, but took my lead from [WAI's Cognitive Accessibility page](https://www.w3.org/WAI/cognitive/), which seems to use consistently use 'cognitive' when preceding the word 'accessibility' but uses 'cognitive and learning' when preceding 'disabilities'.

Reply:
Think I'll leave this for now and we can reassess next time this content is up for review